### ISSUE TITLE
FormInput - Error while updating property 'selectionColor' of a view managed by: AndroidTextInput

### ISSUE THREAD

**Issue by huiming1313** [react-native-training/react-native-elements#423]
I'm using expo 17 and when using FormInput (only in Android), it got error as below.

`<FormInput containerStyle={{ marginBottom:10 }}	placeholder="Name" onChangeText={(val) => this.setState({name: val})} />`

Below is my complete dependecies list

```
{
  "name": "DDC",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "react-native-scripts": "0.0.31",
    "jest-expo": "~1.0.1",
    "react-test-renderer": "16.0.0-alpha.6"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "node node_modules/jest/bin/jest.js --watch"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^17.0.0",
    "react": "16.0.0-alpha.6",
    "react-native": "https://github.com/expo/react-native/archive/sdk-17.0.0.tar.gz",
    "react-native-easy-grid": "^0.1.13",
    "react-native-elements": "^0.12.2",
    "@expo/vector-icons": "^5.0.0",
    "@expo/ex-navigation": "~3.0.0",
    "react-redux": "^5.0.5",
    "react-thunk": "^1.0.0",
    "redux": "^3.6.0",
    "redux-actions": "^2.0.3",
    "redux-thunk": "^2.2.0"
  }
}
```
![untitled](https://user-images.githubusercontent.com/15344977/27144453-7f58ca28-5164-11e7-9676-71ac25ed63d1.png)

**Comment by filiphosko** [comment]
Hi, this should be already fixed in version 0.12.3 as stated here https://github.com/react-native-training/react-native-elements/issues/331. Please try to upgrade your react-native-elements package and let us know if the problem persists (if not, please close this issue :wink: ).

**Comment by Monte9** [comment]
@huiming1313 as @filiphosko mentioned above, this is def fixed. Please update RNE to the latest version.

**Comment by aekold** [comment]
@filiphosko @Monte9 Hi, I still have the problem with the latest 0.17.0 release. The fix #331 is not in the release I don't know why.

**Comment by lejoss** [comment]
im having the same issue, please any update @aekold @filiphosko ?

**Comment by filiphosko** [comment]
@lejoss @aekold You can check out the progress on this here https://github.com/react-native-training/react-native-elements/issues/331#issuecomment-340123332.

**Comment by Monte9** [comment]
@aekold @lejoss please see this comment: https://github.com/react-native-training/react-native-elements/issues/331#issuecomment-340123332

This issue has been fixed and will be released in v0.18 in the next couple days. Stay tuned.
